### ISSUE TITLE
Custom confile file.

### ISSUE THREAD

**Issue by SamMousa** [potsky/PimpMyLog#102]
Currently we can configure the custom configuration dir but not the configuration file name.

Why not use the same structure for that as well? 

It is a minor issue but would make integrating this with other tools easier. 
(I'm building something that has 1 config directory for multiple tools and config.user.php isn't really a descriptive name)

-- I can make a PR if you agree.

**Comment by potsky** [comment]
Hi Sam,

I do not really understand what is the problem.

You can set a `config.user.php` or `config.user.json` file with the globals only and set your software configuration files in `config.user.d` directory with the name you want. Example :

- `config.user.json`
- `config.user.d/apache.json`
- `config.user.d/php.json`
- ...

More informations available here : <http://pimpmylog.com/documentation/configuration.html#toc_25>

Does it make sense ?

**Comment by rhoerbe** [comment]
When you create an docker container (which should be immutable) your should have the software separated from the config. It would therefore be good have an ENV or startup option to point config.user.php outside the software root.

**Comment by rhoerbe** [comment]
I created a docker project working around the issue of config files sitting int he SW root: https://github.com/rhoerbe/docker-pimpmylog

**Comment by potsky** [comment]
It will implemented in version 1.7.11

You just have to add environnement variable `PML_CONFIG_BASE`. In apache for example:

```
rm '/Users/potsky/Work/GitHub/PimpMyLog-dev/_site/inc/test.PLEASE_REMOVE_ME.access_from_1_only.php'
<Directory "/Users/potsky/Sites/">
	Options Indexes MultiViews FollowSymLinks
	Require all granted
	SetEnv PML_CONFIG_BASE /tmp
</Directory>
```
